### PR TITLE
feat: [CDP-20364]: add engineers

### PR DESCRIPTION
**Pull request by allofthesepeople**
- user: nasser.gonzalez@harness.io
- user roger.simms@harness.io
- team CIT

You can use the following comments to re-trigger PR Checks

- Compile: `trigger compile`
- runAeriformCheck: `trigger AeriformCheck`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/30297)

### CODE REVIEW

File: scripts/jenkins/author-check.sh [hunk 1]
```diff
@@ -117,6 +117,7 @@ git log -1000 --oneline --format='%aN <%aE>' | sort -u |\
     grep -iv "^Nataraja Maruthi <nataraja@harness.io>$" |\
     grep -iv "^Nathan Nguyen <nnguyen01@users.noreply.github.com>$" |\
     grep -iv "^Nathan Nguyen <lenathan.nguyen@harness.io>$" |\
+    grep -iv "^Nasser Gonzalez <nasser.gonzalez@harness.io>$" |\
```
Comment (nassergonzalez) [inline scripts/jenkins/author-check.sh:120]:
```suggestion
    grep -iv "^Nasser Gonzalez Gonzalez <nasser.gonzalez@harness.io>$" |\
```

File: scripts/jenkins/authorsList.txt [hunk 2]
```diff
@@ -103,6 +103,7 @@ Mounik V V S S mounik.velavalivenkata@harness.io
 Naman Talaycha naman.talaycha@harness.io
 Naman Verma naman.verma@harness.io
 Nandan Chandrashekar nandan.chandrashekar@harness.io
+Nasser Gonzalez nasser.gonzalez@harness.io
```
Comment (nassergonzalez) [inline scripts/jenkins/authorsList.txt:106]:
```suggestion
Nasser Gonzalez Gonzalez nasser.gonzalez@harness.io
```

**Review by nassergonzalez — APPROVED** [review]
I just added my second surname, apart from that seems good to me

**Review by tmacari-harness — APPROVED** [review]
> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/30297)* status: 3 file(s) to review.

**Review by allofthesepeople — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/30297)* status: 3 file(s) to review. 2. unresolved discussions. 

___
*[scripts/jenkins/author-check.sh, line 120 at r1](https://reviewable.io/reviews/harness/harness-core/30297#-MuQuy5C09-bN3ZlsNZ8-r1-120:-MuV2VtM8F6iND1fo3IJ:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/8e26a7f31469c5ea2b1b56c43417ee90b1c3058d/scripts/jenkins/author-check.sh#L120)):*
<details><summary><i>Previously, nassergonzalez (Nasser Gonzalez Gonzalez) wrote…</i></summary><blockquote>

```suggestion
    grep -iv "^Nasser Gonzalez Gonzalez <nasser.gonzalez@harness.io>$" |\
```
</blockquote></details>

Done.
___
*[scripts/jenkins/authorsList.txt, line 106 at r1](https://reviewable.io/reviews/harness/harness-core/30297#-MuQuy5C09-bN3ZlsNZ9-r1-106:-MuV2WU70gjGmblKpMza:b-896fix) ([raw file](https://github.com/harness/harness-core/blob/8e26a7f31469c5ea2b1b56c43417ee90b1c3058d/scripts/jenkins/authorsList.txt#L106)):*
<details><summary><i>Previously, nassergonzalez (Nasser Gonzalez Gonzalez) wrote…</i></summary><blockquote>

```suggestion
Nasser Gonzalez Gonzalez nasser.gonzalez@harness.io
```
</blockquote></details>

Done.

**Review by nassergonzalez — APPROVED** [review]
> Reviewed 1 of 3 files at r1, 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/30297)* status: 2 file(s) to review.

**Review by tmacari-harness — APPROVED** [review]
> Reviewed 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/30297)* status: :shipit: complete! ✓

**Review by tmacari-harness — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/30297)* status: :shipit: complete! ✓

**Pull request merged**